pages/common/makensis.md

```diff
@@ -0,0 +1,16 @@
+# makensis
+
+> Cross-platform compiler for NSIS installers.
+> It compiles a NSIS script into an Windows installer executable.
```

Comment:
Does this mean that it only works on Windows? Or does it work on all platforms (even though it only spits out installers for Windows)?

Reply:
The compiler itself is available for all POSIX platform, it's only the resulting installer that's for Windows. So adding it to `commons` was a deliberate choice.

Reply:
Ah, excellent! Just wanted to confirm :-)